src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -140,6 +138,17 @@ public void runForIntake( int increPosition) {
 
     }
 
+    public void StartClimberRPM(){
+      if(m_bGoodSensors == true)
+      {
+        m_pidController.setReference(targetRPM, ControlType.kVelocity, 1);
+      }
+      else
+      {
+        m_climber.set(0.2);
```

Comment:
Switch this function so it only uses m_climber.set(0.7) instead of the m_pidController.setReference(...) (just comment it out). 

- .set(0.7) will set 70% motor power which also means 70% of 12V. I am very confidant this will work for North Bay.
- The m_pidController.setReference will use PID to calculate motor power but for North Bay there is no time to tune PID.

D&F plans to put the shooter motor on before drive practice and then climber is ready I think (don't quote me). Would be great to have the drivers try it during drive practice.